@here/harp-mapview/lib/MapView.ts

```diff
@@ -1752,6 +1755,18 @@ export class MapView extends THREE.EventDispatcher {
         return this.m_fog;
     }
 
+    set elevationWrapperThingy(elevationWrapperThingy: ElevationWrapperThingy | undefined) {
+        this.m_elevationWrapperThingy = elevationWrapperThingy;
```

Comment:
If this.m_elevationWrapperThingy was not undefined before we also have to remove the old terrainDataSource.

Reply:
Good point, thanks!